index.ts

```diff
@@ -33,6 +33,14 @@ const settings: SettingSchemaDesc[] = [
       'Enter sync with Omnivore frequency in minutes here or 0 to disable',
     default: 60,
   },
+  {
+    key: 'synced at',
+    type: 'string',
+    title: 'Last Sync',
+    description:
+      'The last time Omnivore was synced. Clear this value to completely refresh the sync.',
+    default: '',
+  },
```

Comment:
We might need to add the validation of the datetime string

Reply:
Good point, i guess if invalid we can just set to an empty string.